src/components/gameCanvas/GameCanvas.tsx

```diff
@@ -4,7 +4,7 @@ import cn from 'classnames';
 
 import GameMain from '../../game/gamemain';
 import { setIsGameStarted, setIsGamePaused, setIsGameOver, setLastScore } from 'store/game/actions';
-import { gameIsGamePausedSelector, gameIsGameOverSelector } from 'store/game/selectors';
+import { gameIsGameStartedSelector, gameIsGamePausedSelector, gameIsGameOverSelector } from 'store/game/selectors';
```

Comment:
🧐
Тут, конечно, нет 120 символов, но строчка получилась длинноватой. Красивее было бы перенести всё руками, но это на твоё усмотрение
```suggestion
import {
  gameIsGameStartedSelector,
  gameIsGamePausedSelector,
  gameIsGameOverSelector
} from 'store/game/selectors';
```